wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -122,6 +126,7 @@ class MainActivity : ComponentActivity() {
     }
 
     private lateinit var application: WalletApplication
+    private val qrEngagementViewModel by viewModels<QrEngagementViewModel>()
```

Comment:
Any reason you are declaring this in a different way from below? Are there any differences? If not, both should probably be declared the same way and I'm also not sure why there's whitespace below.